cmake/cpack_settings.cmake

```diff
@@ -31,4 +31,8 @@ set(CPACK_NUGET_PACKAGE_AUTHORS "Open Enclave SDK Contributors")
 set(CPACK_NUGET_PACKAGE_VERSION ${OE_VERSION})
 set(CPACK_NUGET_PACKAGE_LICENSEURL "https://github.com/openenclave/openenclave/blob/master/LICENSE")
 
+# CPack variables for Nuget packages for the host-side verification library
+set(CPACK_NUGET_OEHOSTVERIFY_PACKAGE_DESCRIPTION "This library is about remote host-side enclave verification. When an application is doing host-side enclave verification, it means that a host application is trying to authenticate a remote enclave's hardware and software settings so that the application can determine whether or not to trust the remote enclave.")
```

Comment:
#9 does this a different way for the sdk packages.

`set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/cmake/NuGetDescription.txt")`

I am wondering if it is best to have this hardcoded in a cmakefile or if it would be better if it was a seperate file that is then sourced?

So maybe something such as

`set(CPACK_NUGET_OEHOSTVERIFY_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/cmake/HostVerifyNuGetDescription.txt")` might be an alternative, to keep the implementation standardized.

Reply:
I tried, but it did not work.

Reply:
Found a workaround by reading a file to a variable first.